hydroflow/src/builder/hydroflow_builder.rs

```diff
@@ -78,6 +80,17 @@ impl HydroflowBuilder {
         (input, pull)
     }
 
+    pub fn add_input_from_stream<T, W, S>(&mut self, stream: S) -> HandoffPullSurface<W>
+    where
+        S: 'static + Stream<Item = T> + Unpin,
+        W: 'static + Handoff + CanReceive<T>,
+    {
+        let (send_port, recv_port) = self.hydroflow.make_edge();
+        self.hydroflow.add_input_from_stream(send_port, stream);
+        let pull = HandoffPullSurface::new(recv_port);
```

Comment:
nit: inline? clippy usually complains to me about this so i dunno why it's ok with this

Reply:
Oh yeah I added `#![allow(clippy::let_and_return)]` a while back because I don't like that lint 😅